.gitattributes

```diff
@@ -1 +1,6 @@
+* text=auto eol=lf
+
+*.AppImage -text
+*.zip -text
```

Comment:
`test/filters/memwrexe` and other in this directory aren't text either.

Reply:
> `test/filters/memwrexe` and other in this directory aren't text either.

Please see the explanation on commit 77e9e4938 (".gitattributes: mark some
binary files as such", 2023-02-20).